### PR TITLE
Bump to 3.2.0

### PR DESCRIPTION
**Pull request by jyemin**

### CODE REVIEW

File: docs/landing/data/releases.toml [hunk 1]
```diff
@@ -1,14 +1,13 @@
 current = "3.1.1"
```
Comment (rozza) [inline docs/landing/data/releases.toml]: current =  "3.2.0"

**Comment by rozza** [comment]
Also, need to update `currentVersion` in `reference/data/mongodb.toml`

File: docs/landing/data/releases.toml (hunk 1 above)
Comment (jyemin) [inline docs/landing/data/releases.toml]: Done

**Comment by jyemin** [comment]
Updated mongodb.toml as well.

**Comment by rozza** [comment]
LGTM :+1:

**Pull request closed without merging**